src/harness/vfsUtil.ts

```diff
@@ -650,14 +650,14 @@ export class FileSystem {
      * NOTE: do not rename this method as it is intended to align with the same named export of the "fs" module.
      */
     public readFileSync(path: string, encoding?: BufferEncoding | null): string | Buffer;
-    public readFileSync(path: string, encoding: BufferEncoding | null = null) { // eslint-disable-line no-null/no-null
+    public readFileSync(path: string, encoding: BufferEncoding | null | undefined = undefined) { // eslint-disable-line no-null/no-null

```

Comment:
It doesn't look like this change is still needed to make this compile.

Reply:
The problem is this overload `public readFileSync(path: string, encoding?: null): Buffer;` I could change that `null` to `undefined`, but I hesitate to change the public API of vfs, although this overload does not appear to be used.

Reply:
I don't quite understand what you mean; I reverted this line to the way it was before and it compiled.

Reply:
Yes. You are right. My bad. It was initially different, but it does work with the original signature now. Fixed